### ISSUE TITLE
Yet another Mono C# MLE misdetection

### ISSUE THREAD

**Issue by Xyene** [DMOJ/judge#437]
See <https://dmoj.ca/submission/1395701>.

Clipped output:

```
Could not allocate 8192 bytes

=================================
```

Verdict: `RTE (write syscall disallowed)`.

This should be mapped to an MLE. Bonus points for figuring out what it's trying to `write` to.

**Comment by ehhthing** [comment]
```
larry@Laptop:/tmp/debug$ mono App.exe
3
2 1 3

Unhandled Exception:
System.FormatException: Input string was not in a correct format.
  at System.Number.ThrowOverflowOrFormatException (System.Boolean overflow, System.String overflowResourceKey) [0x0001a] in <6649516e5b3542319fb262b421af0adb>:0
  at System.Number.ParseInt32 (System.ReadOnlySpan`1[T] value, System.Globalization.NumberStyles styles, System.Globalization.NumberFormatInfo info) [0x00016] in <6649516e5b3542319fb262b421af0adb>:0
  at System.Int32.Parse (System.String s) [0x00016] in <6649516e5b3542319fb262b421af0adb>:0
  at App.Program.Main (System.String[] args) [0x0003e] in <78c70314467d43c3b98ca8c36d533c0a>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.FormatException: Input string was not in a correct format.
  at System.Number.ThrowOverflowOrFormatException (System.Boolean overflow, System.String overflowResourceKey) [0x0001a] in <6649516e5b3542319fb262b421af0adb>:0
  at System.Number.ParseInt32 (System.ReadOnlySpan`1[T] value, System.Globalization.NumberStyles styles, System.Globalization.NumberFormatInfo info) [0x00016] in <6649516e5b3542319fb262b421af0adb>:0
  at System.Int32.Parse (System.String s) [0x00016] in <6649516e5b3542319fb262b421af0adb>:0
  at App.Program.Main (System.String[] args) [0x0003e] in <78c70314467d43c3b98ca8c36d533c0a>:0
```

`strace -o trace.txt -e trace=write mono App.exe`:
```
write(1, "\33[?1h\33=", 7)              = 7
write(1, "\33[6n", 4)                   = 4
write(1, "\33[6n", 4)                   = 4
write(1, "7", 1)                        = 1
write(1, "8", 1)                        = 1
write(1, "4", 1)                        = 1
write(1, "\33[6;3H", 6)                 = 6
write(1, " ", 1)                        = 1
write(1, "\33[6;3H", 6)                 = 6
write(1, "\33[6;2H", 6)                 = 6
write(1, " ", 1)                        = 1
write(1, "\33[6;2H", 6)                 = 6
write(1, "\33[6;1H", 6)                 = 6
write(1, " ", 1)                        = 1
write(1, "\33[6;1H", 6)                 = 6
write(1, "3", 1)                        = 1
write(1, "\n", 1)                       = 1
write(1, "\33[6n", 4)                   = 4
write(1, "1", 1)                        = 1
write(1, " ", 1)                        = 1
write(1, "2", 1)                        = 1
write(1, " ", 1)                        = 1
write(1, "3", 1)                        = 1
write(1, "\n", 1)                       = 1
write(2, "\nUnhandled Exception:\nSystem.For"..., 639) = 639
write(2, "[ERROR] FATAL UNHANDLED EXCEPTIO"..., 652) = 652
write(1, "\33[?1l\33>\33]104\7", 13)    = 13
+++ exited with 1 +++```

**Comment by Xyene** [comment]
This entire class of bugs should be addressed by #446.

**Comment by Xyene** [comment]
#446 is now in master.
